shared/src/commonTest/kotlin/co/nimblehq/mark/kmmic/domain/usecase/GetSurveyUseCaseTest.kt

```diff
@@ -0,0 +1,56 @@
+package co.nimblehq.mark.kmmic.domain.usecase
+
+import co.nimblehq.mark.kmmic.data.service.survey.model.SurveyDetailApiModel
+import co.nimblehq.mark.kmmic.data.service.survey.model.toSurveyDetail
+import co.nimblehq.mark.kmmic.domain.repository.SurveyRepository
+import co.nimblehq.mark.kmmic.dummy.dummy
+import kotlin.test.BeforeTest
+import io.kotest.matchers.shouldBe
+import io.mockative.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.flowOf
+import org.koin.core.context.startKoin
+import org.koin.dsl.module
+import kotlin.test.Test
+import kotlinx.coroutines.test.runTest
+import org.koin.core.context.stopKoin
+import kotlin.test.AfterTest
+
+@ExperimentalCoroutinesApi
+class GetSurveyUseCaseTest {
+
+    private lateinit var useCase: GetSurveyUseCase
```

Comment:
Same rule applies to the property naming.
```suggestion
    private lateinit var getSurveyUseCase: GetSurveyUseCase
```

Reply:
@AVI5HEK updated at [6658248](https://github.com/markgravity/kmm-ic/pull/80/commits/665824802b1e6ab68000900b303fc35b8a2c3fc3)